splice_machine/src/main/java/com/splicemachine/derby/impl/sql/execute/operations/DerbyScanInformation.java

```diff
@@ -461,8 +461,7 @@ public long getConglomerateId() {
         Qualifier[][] scanQualifiers = null;
         if (scanQualifiersFieldName != null) {
             try {
-                if (qualifiersField == null)
-                    qualifiersField = activation.getClass().getField(scanQualifiersFieldName);
```

Comment:
This is also semantically different. Why do we need to overwrite `qualifiersField` when it already holds something?

Reply:
This is restoring the code to the original behavior.  I previously added the null check to avoid some overhead, but I found that qualifiersField is not rebuilt to use the new activation when an operation tree is reused (when we call TableScanOperation.init), and it may point to the wrong row.  We need to build the qualifiersField to refer to the proper activation because we get the row to work on from the row array in the activation.  If we point to the wrong activation, we get the wrong row.  A possible other change could be to rebuild qualifiersField in scanInformation.initialize (called by ScanOperation.init), but it's possible we sometimes need to do more setup before populateQualifiers is called, so it may be risky to move the location of this setup.